src/components/modals/ModalWrapper.vue

```diff
@@ -26,10 +27,20 @@
 export default {
   name: 'ModalWrapper',
 
+  mounted() {
+    this.$refs.closeButton.focus();
+  },
+
   methods: {
     close() {
       this.$emit('close');
     },
-  },
+  }
```

Comment:
Hva skjedde her, er ikke trailing komma skrudd på i prettier-konfigen a? (Som jeg vil slå et slag for at det bør, siden det skaper mindre støy i diffene når man slipper kommaendringer på urelaterte linjer. ;))